Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -8,26 +8,46 @@
 Installation on Google GKE
 **************************
 
+GKE Requirements
+================
+
+1. Install the Google Cloud SDK (``gcloud``) see [Installing Google Cloud SDK](https://cloud.google.com/sdk/install)
+
+2. Create a project or use an existing one
+
+::
+
+   export GKE_PROJECT=gke-clusters
+   gcloud projects create $GKE_PROJECT
+
+
+3. Enable the GKE API for the project if not already done
+
+::
+
+   gcloud services enable --project $GKE_PROJECT container.googleapis.com
+
 Create a GKE Cluster
 ====================
 
 You can apply any method to create a GKE cluster. The example given here is
-using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. 
+using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. This guide
+will create a cluster on zone ``europe-west4-a``, feel free to change the zone
+if you are in a different region of the globe.
 
 .. code:: bash
 
-    gcloud container --project gke-clusters clusters create cluster1 \
-       --username "admin" --image-type COS --num-nodes 2
+    gcloud container --project $GKE_PROJECT clusters create cluster1 \
```

Comment:
Instruct the user to set some environment variable like `GKE_ZONE` instead of hardcoding `europe-west4-a` as I bet some users will blindly copy-and-paste.